src/pages/explore/components/filter/Filter.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import "./filter.css";
+import { useExplore } from "../../../../context/explore/ExploreContext";
+
+export const Filter = () => {
+  const { categories, dispatch, selectedCategory } = useExplore();
+
+  return (
+    <div className="flex-row explore-list-items list-style-none">
+      <button
+        className={`explore-list-item ${
+          selectedCategory.toLowerCase() === "All".toLowerCase() ? "active" : ""
```

Comment:
Suggestion: why not directly giving it a string `all` instead of giving in uppercase and then convert to lowercase.

Reply:
Just for sake of safety check or double check.